File: app/src/main/java/com/ragdroid/dahaka/util/TextUtil.java
package com.ragdroid.dahaka.util;

/**
 * To be able to easily test Presenters.
 *
 * Created by garimajain on 13/08/17.
 */
public class TextUtil {

    private TextUtil() {
        //utility class
    }

    public static boolean isEmpty(String string) {
        return string == null || string.replace(" ", "").length() == 0;
    }

}

File: app/src/main/java/com/ragdroid/dahaka/util/TextUtil.kt
package com.ragdroid.dahaka.util

/**
 * To be able to easily test Presenters.
 *
 * Created by garimajain on 13/08/17.
 */
object TextUtil {

    fun isEmpty(string: String?): Boolean {
        return string == null || string.replace(" ", "").length == 0
    }

}//utility class
